gio/src/auto/dbus_connection.rs

```diff
@@ -798,10 +801,6 @@ impl DBusConnection {
         }
     }
 
-    pub fn get_property_flags(&self) -> DBusConnectionFlags {
```

Comment:
It has a public getter that was added in 2.60. We could add a getter manually for not(v2_60) but that is annoying to do manually... I was thinking of adding that to gir but is it worth the work? 

there are some more cases in a gstreamer crate and also in gtk4 but not that many...

Reply:
this is now fixed